## Упражнения

__Задача 1__

Утверждается, что шарики для подшипников, изготовленные автоматическим станком, имеют средний диаметр 17 мм. Используя уровень значимости 0.025 и правостороннюю критическую область, проверить данную гипотезу при условии, что в выборке из 100 шариков средний диаметр оказался равным 17.5 мм, а дисперсия известна и равна 4 мм.

$\alpha$ = 0.025

$$T(X) = \dfrac{\overline{X} - a}{\sigma / \sqrt{n}} = \dfrac{\overline{X} - 17}{2 / 10} = 5 \cdot (\overline{X} - 17).$$

In [12]:
from scipy import stats
alpha =  0.025
stats.norm.ppf(alpha / 2), stats.norm.ppf(1 - alpha / 2)

(-2.241402727604945, 2.241402727604947)

критическая область: $$\Omega_\alpha = (-\infty, -2.2414) \cup (2.2414, \infty).$$

In [13]:
T = 5 * (17.5 - 17)
T

2.5

2.5 попало в критическую область - гипотеза не верна.

__Задача 2__

Известно, что генеральная совокупность распределена нормально с известным средним квадратическим отклонением 16. Найти доверительный интервал для оценки математического ожидания `a` с надёжностью 0.95, если выборочное среднее равно 80, а объём равен 256.

__Задача 3__

Продавец утверждает, что средний вес пачки печенья составляет 200 г. Из партии извлечена выборка из 10 пачек. Вес каждой пачки составляет:
```
202, 203, 199, 197, 195, 201, 200, 204, 194, 190
```
Известно, что их веса распределены нормально. Верно ли утверждение продавца, если учитывать, что уровень значимости равен 1%?